### PR TITLE
Fix 'nanoid' CVE

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
https://issues.redhat.com/browse/DFBUGS-1193
https://issues.redhat.com/browse/DFBUGS-1194
https://issues.redhat.com/browse/DFBUGS-1195

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -189,7 +189,8 @@
     "webpack-dev-server/express": "^4.21.0",
     "tough-cookie": "^4.1.3",
     "cross-spawn": "^7.0.6",
-    "postcss": "^8.4.49"
+    "postcss": "^8.4.49",
+    "nanoid": "^3.3.8"
```
Comment (alfonsomthd) [inline package.json:193]: We should sort dependencies alphabetically. We can use a plugin to enforce this.

Comment (SanjalKatiyar) [reply package.json:193]: plugin sounds good...

Comment (alfonsomthd) [reply package.json:193]: Can we do it in this PR anyway?

Comment (SanjalKatiyar) [reply package.json:193]: a master only PR would be better IMO, this one needs to be backported till 4.14.z release...

**Pull request merged**